src/client/helpers/utils.tsx

```diff
@@ -226,3 +228,10 @@ export function countWords(text: string) : number {
 	}
 	return words.length;
 }
+
+export const RelationshipTypeEditorIcon = (
+	<span className="fa-layers fa-fw margin-right-0-3">
+		<FontAwesomeIcon icon={faLink} transform="left-3"/>
+		<FontAwesomeIcon icon={faPlus} transform="shrink-8 right-5 down-5"/>
+	</span>
+);
```

Comment:
I love the suggestion for the icon. I was able to create a somewhat similar one using React FontAwesomeIcon as shown here. I have skipped the color styling as we only need a black and white. This is how it looks:
![relationship-type-icon](https://github.com/metabrainz/bookbrainz-site/assets/56965261/b9641219-6ad1-4dc9-8e43-d4b8e46cdd7f)

Reply:
Looking good !